pkg/config/tlscfg/options.go

```diff
@@ -18,6 +18,7 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
+	"github.com/coreos/etcd/pkg/tlsutil"
```

Comment:
I don't think we want an external dependency for such minor functionality. Also, I am not seeing this package in https://github.com/etcd-io/etcd/tree/main/pkg

Reply:
@yurishkuro removed this external dependency and made it a sample. Hope this makes sense.

Reply:
as it was mentioned in this document  https://docs.fluentd.org/configuration/transport-section#tls-setting do you want to provide some default cipher in the repo itself?

Reply:
and one more doubt why naming ciphers if want to pass just string ? are we expecting multiple ciphers ?